lib/provider.js

```diff
@@ -352,12 +353,22 @@ export default function() {
             // $rootScope.$broadcast('ionicNativeTransitions.', executePendingTransition);
             registerToStateChangeStartEvent();
         }
+        
+        function disablePendingTransition() {
+            // If native transition support cancelling transition (> 0.6.4), cancel pending transition
+            if (angular.isFunction(window.plugins.nativepagetransitions.cancelPendingTransition)) {
```

Comment:
I think here we should also check for the plugins and nativetransitions otherwise we will get `Uncaught ReferenceError`

Something like this:

```
window.plugins && window.plugins.nativepagetransitions && angular.isFunction(window.plugins.nativepagetransitions.cancelPendingTransition)
```